go.mod

```diff
@@ -4,10 +4,10 @@ go 1.12
 
 require (
 	github.com/containernetworking/cni v0.7.2-0.20190904153231-83439463f784
-	github.com/containers/common v0.20.3
+	github.com/containers/common v0.20.3-0.20200827091701-a550d6a98aa3
```

Comment:
@rhatdan just bumped to v0.21.0, would you mind changing this to that please @saschagrunert ?

Reply:
Sure, done :+1: